fabric/src/main/java/com/discordsrv/fabric/module/chat/FabricJoinModule.java

```diff
@@ -72,11 +73,24 @@ private void onJoin(ServerPlayNetworkHandler serverPlayNetworkHandler, PacketSen
     private MinecraftComponent getJoinMessage(ServerPlayerEntity playerEntity) {
         MutableText mutableText;
         if (playerEntity.getGameProfile().getName().equalsIgnoreCase(playerEntity.getName().getString())) {
+            //? if minecraft: <1.19 {
+            /*mutableText = new net.minecraft.text.TranslatableText("multiplayer.player.joined", playerEntity.getDisplayName());
+            *///?} else {
             mutableText = Text.translatable("multiplayer.player.joined", playerEntity.getDisplayName());
+             //?}
         } else {
+            //? if minecraft: <1.19 {
+            /*mutableText = new net.minecraft.text.TranslatableText("multiplayer.player.joined.renamed", playerEntity.getDisplayName(), playerEntity.getName());
+            *///?} else {
             mutableText = Text.translatable("multiplayer.player.joined.renamed", playerEntity.getDisplayName(), playerEntity.getName());
+             //?}
```

Comment:
Is it possible to convert all these translatable components to use Adventure (`Component.translatable(...)`) directly?